src/main/java/net/sf/jabref/importer/fileformat/JSONEntryParser.java

```diff
@@ -0,0 +1,150 @@
+/*  Copyright (C) 2015 Oscar Gustafsson.
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+package net.sf.jabref.importer.fileformat;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import net.sf.jabref.model.entry.BibtexEntry;
+import net.sf.jabref.model.entry.BibtexEntryTypes;
+
+public class JSONEntryParser {
+
+    private static final Log LOGGER = LogFactory.getLog(JSONEntryParser.class);
+
+
+    public JSONEntryParser() {
+
+    }
+
+    public BibtexEntry BibJSONtoBibtex(JSONObject bibJsonEntry) {
+        // Fields that are directly accessible at the top level BibJson object
+        String[] singleFieldStrings = new String[] {"year", "title", "abstract", "month"};
+
+        // Fields that are accessible in the journal part of the BibJson object
+        String[] journalSingleFieldStrings = new String[] {"publisher", "number", "volume"};
```

Comment:
You can just write 

`String[] journalSingleFieldStrings = {"publisher", "number", "volume"};`

instead of 

`String[] journalSingleFieldStrings = new String[] {"publisher", "number", "volume"};`